src/Library/demos/Box/main.blp

```diff
@@ -0,0 +1,109 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Box";
+      margin-top: 12;
+      margin-bottom: 12;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: "A widget that arranges its child widgets into a single row or column";
+      margin-bottom: 18;
+    }
+
+    Box {
+      orientation: vertical;
+
+      Box {
+        hexpand: true;
+        orientation: vertical;
+        spacing: 12;
+
+        Box {
+          halign: center;
+          spacing: 18;
+          Button button_add{
+            label: "Add Item";
+            styles [
+              "suggested-action",
+              "pill",
+            ]
+          }
+          Button button_remove{
+            label: "Remove Item";
+            styles [
+              "destructive-action",
+              "pill",
+            ]
+          }
+        }
+
+        Box {
+          halign: center;
+          spacing: 18;
+
+          CheckButton highlight {
+            label: "Highlight Box";
+          }
+
+          CheckButton toggle_orient {
+            label: "Toggle Orientation";
+          }
+
+          CheckButton homogeneous {
+            label: "Homogeneous";
+          }
+
+          CheckButton prepend {
+            label: "Prepend Items";
+          }
+        }
+
+        Label {
+          label: "Horizontal Alignment";
+        }
+
+        Scale alignment {
+            margin-bottom: 18;
+            adjustment: Gtk.Adjustment {
+              lower: 0;
+              upper: 100;
+              value: 0;
+            };
+        }
+      }
+
+      ScrolledWindow {
```

Comment:
```suggestion
      ScrolledWindow {
        has-frame: true;
```

Add a frame to this, so it's clear how the box is moving in relation to its parent.

There used to be this helpful image in our documentation, with each tile showing the widget's geometry and position in relation to the parent. So as you're flipping the switches in your demo, each combination would show one of the grid tiles below:

![widget-hvalign](https://user-images.githubusercontent.com/1265208/224516542-41a13313-8c8b-49c2-87d3-14e8a35c4eff.png)